src/Auth.UI.WinUI3/Auth.UI.WinUI3.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```

Comment:
How did you choose these versions, are they reasonable? (I'm not so familiar with Win10 version, forgive my dumb question :) )

Reply:
They are the default values for WinUI3 now. 
It means that it will work on all versions of Windows 11 and Windows 10 starting from  Windows 10, version 1809 - that includes most of Windows 10 systems.